packages/web-react/src/components/ButtonLink/README.md

```diff
@@ -17,22 +17,48 @@ import { ButtonLink } from '@lmc-eu/spirit-web-react';
-| `isLoading`     | `bool`                                                                                        | `false`   | ✕        | If true, ButtonLink is in a loading state, disabled and the Spinner is visible |
-| `isSymmetrical` | `bool`                                                                                        | `false`   | ✕        | If true, ButtonLink has symmetrical dimensions, usually only with an Icon      |
-| `onClick`       | `string`                                                                                      | `null`    | ✕        | JS function to call on click                                                   |
-| `ref`           | `ForwardedRef<HTMLAnchorElement>`                                                             | —         | ✕        | Anchor element reference                                                       |
-| `size`          | [Size dictionary][dictionary-size]                                                            | `medium`  | ✕        | Size variant                                                                   |
-| `target`        | `string`                                                                                      | `null`    | ✕        | Link target                                                                    |
+| Name            | Type                                                                                          | Default   | Required | Description                                                                            |
+| --------------- | --------------------------------------------------------------------------------------------- | --------- | -------- | -------------------------------------------------------------------------------------- |
+| `children`      | `ReactNode`                                                                                   | `null`    | ✕        | Content of the ButtonLink                                                              |
+| `color`         | [Component Button dictionary][dictionary-color], [Emotion Color dictionary][dictionary-color] | `primary` | ✕        | Color variant                                                                          |
+| `elementType`   | `ElementType`                                                                                 | `a`       | ✕        | Type of element                                                                        |
+| `href`          | `string`                                                                                      | —         | ✓        | Link URL                                                                               |
+| `isBlock`       | `bool`                                                                                        | `false`   | ✕        | [**DEPRECATED**](#deprecation-notice) Span the element to the full width of its parent |
```

Comment:
i would link the section too. For lazy devs that only read the API table. See Button.